ads/google/doubleclick.js

```diff
@@ -253,45 +229,19 @@ function centerAd(global) {
   });
 }
 
-/**
- * @param {!Object} data
- * @return {!string}
- */
-export function selectGptExperiment(data) {
-  const fileExperimentConfig = {
-    21060540: 'gpt_sf_a.js',
-    21060541: 'gpt_sf_b.js',
-  };
-  // Note that reduce will return the first item that matches but it is
-  // expected that only one of the experiment ids will be present.
-  let expFilename;
-  (data['experimentId'] || '').split(',').forEach(
-      val => expFilename = expFilename || fileExperimentConfig[val]);
-  return expFilename;
-}
-
 /**
  * @param {!Window} global
  * @param {!Object} data
- * @param {!string} gptFilename
  */
-export function writeAdScript(global, data, gptFilename) {
-  const url =
-  `https://www.googletagservices.com/tag/js/${gptFilename || 'gpt.js'}`;
-  if (gptFilename || data.useSameDomainRenderingUntilDeprecated != undefined
-    || data.multiSize) {
-    doubleClickWithGpt(global, data, GladeExperiment.GLADE_OPT_OUT, url);
+export function writeAdScript(global, data) {
+  if (data.useSameDomainRenderingUntilDeprecated != undefined
+      || data.multiSize || (
+          global.context && global.context.location &&
+            global.context.location.href &&
+            global.context.location.href.indexOf('google_glade=0') > 0 &&
```

Comment:
This really should have used location.search as well as a regexp extract IMO but I suppose you're just copying over existing...